### ISSUE TITLE
"Failed to install Plugin Smart connections"

### ISSUE THREAD

**Issue by Akimika2034** [brianpetro/obsidian-smart-connections#439]
"I tried to install the plugin, but this message appears immediately :"Failed to install Plugin Smart connections" . Can someone please help me?

**Comment by brianpetro** [comment]
Hey @Akimika2034 

Can you try to install again now?

Had a few hiccups earlier, but I believe it's working now.

Let me know.

Thanks for bringing this to my attention!

Brian 🌴

**Comment by 360-JU57US** [comment]
I'm having the same issue today, March 23, 2025.

**Comment by brianpetro** [comment]
@360-JU57US can you screenshot any errors you see in the developer console logs and share them here so I can check it out? 🌴

**Comment by brianpetro** [comment]
@360-JU57US I think the issue might of been https://github.com/brianpetro/obsidian-smart-connections/issues/1015#issuecomment-2746429316 which was just resolved. let me know if it starts working for you now 🌴

**Comment by 360-JU57US** [comment]
> [@360-JU57US](https://github.com/360-JU57US) I think the issue might of been [#1015 (comment)](https://github.com/brianpetro/obsidian-smart-connections/issues/1015#issuecomment-2746429316) which was just resolved. let me know if it starts working for you now 🌴

Yes!  I manually installed 2.5.7 as a workaround, and then several hours later I tried updating and it worked fine.  Thanks!

**Comment by brianpetro** [comment]
Closing since haven't heard any updates from @Akimika2034 🌴
